torch/_inductor/ir.py

```diff
@@ -6863,6 +6863,232 @@ def create(
         )
 
 
+class QLinearPointwiseBinaryPT2E(ExternKernelAlloc):
```

Comment:
Hi @eellison Thanks for reviewing. Compared with the kernel for unary post ops, the kernel for binary post ops has additional arguments, different schemas, and different codegen methods, etc. So, it will be clearer to separate binary with unary classes. Do you think it's ok?

Reply:
Oh, the primary reason is that we have to handle the mutation semantics on these ops, i.e., the binary post-op supports the in-place writes into the other input. I guess this is not supported by the implicit fallbacks.

Reply:
You could override the `get_mutation_names` on the kernel instead ?

Reply:
Hi @eellison This class is indeed a subclass of the Extern kernel. Did you suggest merging `QLinearPointwisePT2E`, which we already have, and `QLinearPointwiseBinaryPT2E`? If not, could you please advise how to implement this? Are there any examples? Thanks.

Reply:
Hi @eellison Could you please take a look and comment? Thanks.

Reply:
Not specific to this PR, but can we move all the x86 specific IRs into a separate file? The file has been growing too big.

Reply:
Thanks for the suggestions @desertfire. Will follow up this one.